plugins/query-insights/src/test/java/org/opensearch/plugin/insights/QueryInsightsPluginTests.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.insights;
+
+import org.opensearch.test.OpenSearchTestCase;
+import org.junit.Assert;
+
+public class QueryInsightsPluginTests extends OpenSearchTestCase {
+
+    public void testDummy() {
```

Comment:
Curious why this is needed?

Reply:
This was a placeholder, let me add more unit tests here.